deploy/21_hlg.ts

```diff
@@ -65,14 +65,28 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     process.exit();
   };
 
+  let tokenAmount: BigNumber = BigNumber.from('100000000');
```

Comment:
Lets make this and the same code in `02_sources.ts` a function to protect ourselves from randomly modifying it.